### PR TITLE
Lucina Testnet Intructions

### PR DESCRIPTION
**Pull request by JakeHartnell**
Hopefully these make sense. Reorganized things a bit.

Validators will submit PRs to this repo with their gentx.

### CODE REVIEW

File: STARPORT.md [hunk 1]
```diff
@@ -32,6 +36,32 @@ starport network proposal approve [chainID] 1,4,5,6 --nightly
...
+
+**IMPORTANT: Be sure to run the following on the machine you'll use for the testnet.**
+
+Run the following command from a server to propose yourself as a validator:
+
+```
+starport network chain join [chainID] --nightly
+```
+
+Follow the prompts to provide information about the validator. Starport will download the source code of the blockchain node, build, initialize and create and send two proposals to SPN: to add an account and to add a validator with self-delegation. By running a `join` command you act as a "validator". When filling out the required parameters ensure to include the **'stake'** word after the required values for the inputs to be accepted. If the terminal gets an error or hangs then you can also try: `starport network chain join [chainID] --nightly --keyring-backend "test"`
```
Comment (blockcreators) [inline STARPORT.md:49]: , initialize and create and send two proposals could be changed to:
, initialize, create and send two proposals

File: lucina/README.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+- 25GB of disk space
+- 1.4 GHz CPU
+
+The first testnet will test using SPN to deploy the chain and connect validators.
+
+The goal is simply to get the chain started and assess the viability of using SPN.
+
+### Parameters
+
+- _Start time: TBD_
```
Comment (blockcreators) [inline lucina/README.md:67]: Should we add this now?

**Review by blockcreators — CHANGES_REQUESTED** [review]
two comments

File: lucina/README.md (hunk 2 above)
Comment (JakeHartnell) [reply lucina/README.md]: Added at the top of the page.

**Pull request merged**